lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_lowering_context.h

```diff
@@ -33,10 +33,12 @@ class GenericComputationTS : public lazy_tensors::GenericComputation {
                                       lazy_tensors::Shape());
   }
 
-  std::shared_ptr<torch::jit::Graph> graph() const { return graph_; }
+  std::shared_ptr<torch::jit::Graph> GetGraph() const { return graph_executor_.graph(); }
```

Comment:
Doesn't seem to be needed to pass a shared_ptr back if the above is the only call site.
```
return ts_computation->GetGraph()->toString();
```